### ISSUE TITLE
"Metadata:Created at" not autopopulating in QGIS 3.20.2 (but is in QGIS 3.18.0)

### ISSUE THREAD

**Issue by everydayduffy** [3liz/qgis-pgmetadata-plugin#75]
Hello,

Firstly, thank you very much for making a very useful and well designed QGIS plugin!

When using the plugin on QGIS 3.20.2 the "Created At" field in Metdata will not autopopulate and therefore not allow me save the edits.

![image](https://user-images.githubusercontent.com/15983662/131655192-5f136ab1-0716-4c24-8e93-6448b9eb3cef.png)

In comparison, loading the exact same administrator project from the same database in QGIS 3.18.0 causes no issues.

Given it is the more recent version that is misbehaving, I thought I would let you know.

Thanks.

**Comment by everydayduffy** [comment]
In the process of updating from 3.20.0 to 3.20.2 I think I updated some dependencies... 3.18.0 then had the same issue with the metadata. 

I uninstalled both, and from scratch installed the 3.16 LTR version with OSGEO4W. The same issue persists. Here is a screenshot of my "About" Info:

![image](https://user-images.githubusercontent.com/15983662/131667755-d39b5409-2ca5-4f6a-8735-fff64bc8c0ae.png)

**Comment by Gustry** [comment]
Can you update the QGIS project, layer properties of `Dataset`, `Forms` for at least the "Created" field ?

![Screenshot from 2021-09-01 14-18-57](https://user-images.githubusercontent.com/1609292/131670012-aa621153-8949-4c1f-8f38-ee537d87c755.png)

I don't understand why we don't have a default value set, while we have a constraint.

```
diff --git a/pg_metadata/resources/projects/pg_metadata_administration.qgs b/pg_metadata/resources/projects/pg_metadata_administration.qgs
index a0abb27..0b4d850 100644
--- a/pg_metadata/resources/projects/pg_metadata_administration.qgs
+++ b/pg_metadata/resources/projects/pg_metadata_administration.qgs
@@ -1060,8 +1060,8 @@ def my_form_open(dialog, layer, feature):
         <default applyOnUpdate="0" expression="" field="projection_name"></default>
         <default applyOnUpdate="0" expression="" field="projection_authid"></default>
         <default applyOnUpdate="0" expression="" field="spatial_extent"></default>
-        <default applyOnUpdate="0" expression="" field="creation_date"></default>
-        <default applyOnUpdate="0" expression="" field="update_date"></default>
+        <default applyOnUpdate="0" expression="now()" field="creation_date"></default>
+        <default applyOnUpdate="1" expression="now()" field="update_date"></default>
         <default applyOnUpdate="0" expression="" field="data_last_update"></default>
         <default applyOnUpdate="0" expression="" field="themes"></default>
       </defaults>
```

**Comment by everydayduffy** [comment]
Thanks for the quick response. Manually setting the values of both creation_date and update_date to `now()` (and checking `apply to value on update` for update_date, then saving the project has worked. I tried closing Q and reloading and those settings have stayed in the project. 

I think it's fixed for me. Thank you.

**Comment by Gustry** [comment]
This needs to be fixed in plugin.

You can see the discussion about this fix in #77
